src/Details/searchDetail.tsx

```diff
@@ -0,0 +1,28 @@
+import { useSelector } from "react-redux"
+import { useParams } from "react-router-dom"
+import { Link } from "react-router-dom"
+import { VscChevronLeft } from 'react-icons/vsc'
+
+export function SearchDetail(){
+  const News:{}[] = useSelector((prev: any)=> prev.search.value)
```

Comment:
Could you create a type for a single News article rather than an empty object type?